python/ray/experimental/workflow/examples/comparisons/google_cloud_workflows/sub_workflows_workflow.py

```diff
@@ -0,0 +1,23 @@
+from ray import workflow
+
+
+@workflow.step
```

Comment:
I feel `sub_workflows_google.yaml` actually is a nested workflow.

They have 
```
step1 -> step2
  |
hello
```

Here hello is another workflow.

Reply:
Hmm I thought sub==nested, do you have an alternate suggestion on how to implement the example?